modules/local/callingcardstools/mammals/concatQC/environment.yml

```diff
@@ -1,3 +1,4 @@
+name: callingcardstools
```

Comment:
shouldn't it be the name of the module here rather than the pipeline?

Reply:
This was per @maxulysse . From him, my understanding that is the name of the environment. Since I am using the same environment for all of the `callingcardstools` modules, by using the same `name` in each, it means that if the env exists it won't be downloaded again by a subsequent module.

Reply:
if @maxulysse says so I'm ok with that